src/data/sql/brk2/select.kadastraal_subject.sql

```diff
@@ -156,6 +158,5 @@ FROM subjecten sjt
          LEFT JOIN brk2.c_land lbu ON (obu.land_code = lbu.code)
          LEFT JOIN brk2.c_land pbu ON (pau.land_code = pbu.code)
          LEFT JOIN brk2_prep.subject_expiration_date ede ON sjt.Identificatie_subject = ede.subject_id
-         LEFT JOIN brk2.bsn_overleden bon ON (sjt.Heeft_BSN_voor = bon.bsn)
          JOIN brk2_prep.meta meta ON TRUE
```

Comment:
nu je de code toch aanraak, zou je deze regel `JOIN brk2_prep.meta meta ON TRUE` kunnen verwijderen?
meta tabel wordt niet meer gebruikt in de select

Reply:
zal ik doen